weeks/week07/week07.md

```diff
@@ -1,3 +1,91 @@
 # Week 7: Web Automation with Playwright
 
-Coming soon.
+## Playwright general information:
```

Comment:
There are two heading with almost the same title ("Playwright general information" and "Playwright - Python General information:"). Try to summarize both a summary/introduction, right after the H1 ("Week 7: ...")